v20.2/alter-column.md

```diff
@@ -59,6 +59,8 @@ For an example of `ALTER COLUMN TYPE`, see [Increase a column type's precision](
 
 ## Examples
 
+>ALTER TABLE student ALTER clg_fee type DECIMAL (12,2);
```

Comment:
To be consistent with the other examples on this page, your example should have some text explaining exactly what the example is doing.

For reference, see the example with the title `### Set or change a DEFAULT value`.